src/python/notation.py

```diff
@@ -669,8 +669,13 @@
     "Decode doc ids in BKD leaves with auto-vectorized loops. https://github.com/apache/lucene/pull/14203",
   ),
   (
-    "2025-03-20",
```

Comment:
Hmm is it intentional that you removed this annotation?

Reply:
Oops sorry, not intended, fixed.